_sass/_hackathon.scss

```diff
@@ -0,0 +1,24 @@
+#csh-hacks-content{
+  padding-top: 10px;
+  padding-bottom: 10px;
+  & h2 {
+    color: white;
+  }
+  & img{
```

Comment:
The file still has lots of incorrect formatting, like `img{` vs `img {`, please install a linter like Prettier in your editor or otherwise run through and make sure the code is formatted correctly. I.e. one space between selector names + braces, and single newlines between blocks.